### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by olya-yevsieienko**

### CODE REVIEW

**Review by IvanGrabovsky — APPROVED** [review]
good job

File: src/herbivoresAndCarnivores.js [hunk 1]
```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
```
Comment (jstmpelowycz) [inline src/herbivoresAndCarnivores.js:4]: you don't need `health`, check the examples of how it should work

Comment (jstmpelowycz) [reply src/herbivoresAndCarnivores.js:4]: also, you don't even use it, just remove

File: src/herbivoresAndCarnivores.js [hunk 2]
```diff
@@ -1,15 +1,40 @@
...
 class Carnivore extends Animal {
-  // write your code here
+  bite(prey) {
+    if (prey instanceof Carnivore || prey.hidden) {
+      return;
+    }
+
+    prey.health -= 50;
+
+    if (prey.health === 0) {
```
Comment (jstmpelowycz) [inline src/herbivoresAndCarnivores.js:34]:
```suggestion
    if (!prey.health) {
```

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**